js/modules/k6/http/http.go

```diff
@@ -129,6 +131,8 @@ func (*HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.State,
 	}
 	if userAgent := state.Options.UserAgent; userAgent.Valid {
 		req.Header.Set("User-Agent", userAgent.String)
+	} else {
+		req.Header.Set("User-Agent", fmt.Sprintf("k6/%s (https://k6.io/);", version.Version))
```

Comment:
Please make this a default for the flag instead.

Reply:
@liclac There's no cli flag for the user agent, are you suggesting we add one?

Reply:
Yeah. There's one in the cobra branch and I sometimes forget where the differences are...

Reply:
@liclac Ok, I'll wait until the cobra branch is merged then, and then implement this on top of that.